media/src/main/webapp/app-root/clientlibs/authoring/dialog/js/fileupload.js

```diff
@@ -87,6 +87,12 @@
    * Trigger 'assetselected' event on the fileupload widget.
    */
   FileUploadExtension.prototype._triggerAssetSelected = function (assetPath) {
+
+    // special case when asset path is removed in the pathfield we treat it as click on the clear button for file upload
+    if (assetPath === "") {
+      this._$element.find("[coral-fileupload-clear]").trigger($.Event("click"));
```

Comment:
we should leave the methods in this case - otherwise the following code lines generate invalid image URLs with an empty asset path.
or probably better: introduce a different method "_triggerAssetClear" that is called instead when the asset path is empty.